datajoint/erd.py

```diff
@@ -208,7 +208,10 @@ def _make_graph(self):
             graph = nx.DiGraph(self).subgraph(nodes)
             nx.set_node_attributes(graph, 'node_type', {n: _get_tier(n) for n in graph})
             # relabel nodes to class names
-            mapping = {node: (lookup_class_name(node, self.context) or node) for node in graph.nodes()}
+            clean_context = dict((k, self.context[k]) for k in self.context
+                                 if '_' not in k)  # hack for ipython '_' var
```

Comment:
shouldn't this be 
`if k != '_'`  or `if not k.beginswith('_')`?

Reply:
Why are we rejecting everything containing `_`? I think this should be `k.beginswith('_')`